webpack.config.js

```diff
@@ -0,0 +1,50 @@
+const webpack = require('webpack');
```

Comment:
webpack is imported here, but I don't see it used explicitly.

Reply:
@Gledinburgh Nice catch again!! Im not exactly sure as to why that is the case. I am thinking that the config object that is exported from the file is used with "webpack" somewhere behind the scenes when the transpiling and bundling is taking place.